app/services/payout_report_publisher_includer.rb

```diff
@@ -10,40 +10,52 @@ def perform
     publisher_has_unsettled_balance = false
 
     wallet = @publisher.wallet
-    # If the wallet has been blocked from uphold's terms of service. In this state users are unable to login or access uphold.
-    return if wallet.nil? || wallet.blocked?
 
+    uphold_status = wallet.uphold_account_status
+    uphold_member = wallet.is_a_member?
+    uphold_connected = wallet.uphold_id.present? # TODO consider adding sub category for when wallet.action == 're-authorize'
```

Comment:
I realized this is not needed and can also be expressed as `uphold_account_status == nil`, I will track reauthorization need instead.